common/src/java/com/zimbra/common/util/Log.java

```diff
@@ -36,23 +38,23 @@ public class Log {
 
     private final Map<String, Logger> mAccountLoggers = new ConcurrentHashMap<String, Logger>();
 
-    private static final Map<Level, org.apache.log4j.Level> ZIMBRA_TO_LOG4J =
-        new EnumMap<Level, org.apache.log4j.Level>(Level.class);
+    private static final Map<Level, org.apache.logging.log4j.Level> ZIMBRA_TO_LOG4J =
+            new EnumMap<Level, org.apache.logging.log4j.Level>(Level.class);
     static {
-        ZIMBRA_TO_LOG4J.put(Level.error, org.apache.log4j.Level.ERROR);
-        ZIMBRA_TO_LOG4J.put(Level.warn, org.apache.log4j.Level.WARN);
-        ZIMBRA_TO_LOG4J.put(Level.info, org.apache.log4j.Level.INFO);
-        ZIMBRA_TO_LOG4J.put(Level.debug, org.apache.log4j.Level.DEBUG);
-        ZIMBRA_TO_LOG4J.put(Level.trace, org.apache.log4j.Level.TRACE);
-    }
-    private static final Map<org.apache.log4j.Level, Level> LOG4J_TO_ZIMBRA =
-        new ImmutableMap.Builder<org.apache.log4j.Level, Level>()
-        .put(org.apache.log4j.Level.FATAL, Level.error)
-        .put(org.apache.log4j.Level.ERROR, Level.error)
-        .put(org.apache.log4j.Level.WARN, Level.warn)
-        .put(org.apache.log4j.Level.INFO, Level.info)
-        .put(org.apache.log4j.Level.DEBUG, Level.debug)
-        .put(org.apache.log4j.Level.TRACE, Level.trace)
+        ZIMBRA_TO_LOG4J.put(Level.error, org.apache.logging.log4j.Level.ERROR);
```

Comment:
looks like we are missing FATAL level error here in the update

Reply:
No.. We are mapping only org.apache.log4j.Level.FATAL to Level.error not vice versa. We were doing the same earlier too.

Reply:
right, but it was there as a different level though it was mapped to Level.error. I think we should keep it the same.

Reply:
got it, you are right. I miss read it.